### DISCUSSION TITLE
The translation of website's some pages from English to Chinese is a mess?

### DISCUSSION THREAD

**Discussion by JamesTaylor04** [kubernetes/website#43751] [General]
Please make translation better, thank you!

**Comment by divya-mohan0209** [comment]
@JamesTaylor04 : Thank you for your concern! The Kubernetes project and its documentation is maintained by voluntary contributions. If there's a specific page or set of pages that you think are translated incorrectly, please feel free to submit a change via a pull request to the repository or open an issue specifying **more details**. Opening discussion items is unhelpful because we lack context about which pages you're referring to exactly. Therefore, I'll be closing & locking this conversation.
